src/home_robot/home_robot/motion/stretch.py

```diff
@@ -61,12 +65,13 @@
         0.01,  # arm
         0.0,  # gripper rpy
         0.0,  # wrist roll
-        0.0,  # wrist pitch
+        -0.7,  # wrist pitch
```

Comment:
Will we be doing this for both the navigations? Will remember to do this in sim. May protrude scene in rare cases but gripper down with object held may not be practical.

Reply:
I think we need to do this, yes; otherwise it will hit stuff when it moves right?